dotcom-rendering/makefile

```diff
@@ -67,12 +67,12 @@ run-ci: stop build-ci start-ci
 # dev #########################################
 
 dev: clear clean-dist install
-	$(call log, "starting frontend DEV server")
-	@NODE_ENV=development SKIP_LEGACY=true nodemon scripts/dev-server/index -- --max-old-space-size=7168
+	$(call log, "starting DEV server")
+	@NODE_ENV=development SKIP_LEGACY=true webpack serve --config ./scripts/webpack/webpack.config.js
 
 dev-legacy: clear clean-dist install
-	$(call log, "starting frontend DEV server (Legacy support ENABLED)")
-	@NODE_ENV=development nodemon scripts/dev-server/index -- --max-old-space-size=7168
+	$(call log, "starting DEV server")
```

Comment:
Why drop `(Legacy support ENABLED)` ?

Reply:
if you run in legacy mode, you can see it bundling as start/save changes

<img width="309" alt="image" src="https://user-images.githubusercontent.com/867233/153465775-cd19201a-2bff-43fd-b68e-aba1b4500977.png">